src/content/updates/v10-migration/designers.mdx

```diff
@@ -141,21 +120,21 @@ Any icons being used with Carbon v9 (outside of those integral to a component) s
 [Iconography guidelines](/guidelines/iconography/library)
```

Comment:
The links aren't stacking. Need a empty line break between links. (That's left over my my issue, sorry)
![image](https://user-images.githubusercontent.com/11670886/55261409-dfece000-5238-11e9-99fd-fdedb17a5443.png)

Reply:
Whoops, this wasn't in my local build. Thanks for pointing this out!